manager/src/main/java/org/apache/hertzbeat/manager/controller/PluginController.java

```diff
@@ -85,4 +87,19 @@ public ResponseEntity<Message<Void>> updatePluginStatus(@RequestBody PluginMetad
         pluginService.updateStatus(plugin);
         return ResponseEntity.ok(Message.success("Update success"));
     }
+
+    @GetMapping("/getParamDefine")
+    @Operation(summary = "get param define", description = "get param define by jar path")
+    public ResponseEntity<Message<PluginParametersVO>> getParamDefine(@RequestParam Long pluginMetadataId) {
+        PluginParametersVO plugins = pluginService.getParamDefine(pluginMetadataId);
+        return ResponseEntity.ok(Message.success(plugins));
+    }
+
+    @PostMapping("/saveParams")
```

Comment:
recommend to use all lowercase letters in the URL and use http method to indicate the action like `/params/define`